homeassistant/components/lcn/climate.py

```diff
@@ -72,42 +86,42 @@ def __init__(self, config, entry_id, device_connection):
         self._target_temperature = None
         self._is_on = True
 
-    async def async_added_to_hass(self):
+    async def async_added_to_hass(self) -> None:
         """Run when entity about to be added to hass."""
         await super().async_added_to_hass()
         if not self.device_connection.is_group:
             await self.device_connection.activate_status_request_handler(self.variable)
             await self.device_connection.activate_status_request_handler(self.setpoint)
 
-    async def async_will_remove_from_hass(self):
+    async def async_will_remove_from_hass(self) -> None:
         """Run when entity will be removed from hass."""
         await super().async_will_remove_from_hass()
         if not self.device_connection.is_group:
             await self.device_connection.cancel_status_request_handler(self.variable)
             await self.device_connection.cancel_status_request_handler(self.setpoint)
 
     @property
-    def supported_features(self):
+    def supported_features(self) -> int:
         """Return the list of supported features."""
         return const.SUPPORT_TARGET_TEMPERATURE
 
     @property
-    def temperature_unit(self):
+    def temperature_unit(self) -> str:
         """Return the unit of measurement."""
-        return self.unit.value
+        return str(self.unit.value)
```

Comment:
Instead of this, cast to a string.
Also, the pypck lib could be updated to hint that the `VarUnit` values are all strings.

Reply:
Noted. Unfortunately VarUnit is an `Enum` and I have to get familiar with typing for `Enum` values.
As soon as I get it implemented in `pypck` I will remove the casts here.